scripts/10x_chromium_hashtag_csv.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+DESC = """This script is intended to run as a project automation,
+taking a CSV whose columns are sample names and semi-colon separated
+antibody product numbers.
+
+Generated at https://genomics-status.scilifelab.se/10X_chromium_hashtag_csv
+
+Example CSV "P31907_20250403_1334.csv"
+
+    Sample_ID,Antibodies
+    P31907_1001,394663;682205;682207
+    P31907_1002,394667;394669
+    P31907_1003,394669
+    P31907_1004,
+
+The script will scan the project files for such a csv, and upon finding it will
+write the 2nd column to the matching sample's UDF "Antibodies".
+"""
+
+import re
+import sys
+from argparse import ArgumentParser
+
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Project
+from genologics.lims import Lims
+
+
+def main(args):
+    lims = Lims(BASEURI, USERNAME, PASSWORD)
+    proj = Project(lims, id=args.proj_id)
+
+    antibody_csv_pattern = re.compile(proj.id + r"_\d{8}_\d{4}\.csv")
```

Comment:
Check whether we want more specific pattern

Reply:
I think it's fine since we cover the cases where more than one file is found already. Can be fixed in the future if needed.